README.md

```
@@ -10,6 +10,23 @@ This repository is linked to a [v0](https://v0.app) project. You can continue de
 
 ## Getting Started
 
+### Environment Setup
+
+1. Copy the environment variables template:
+   ```bash
+   cp .env.example .env
+   ```
+
+2. Fill in your Ozon API credentials in `.env`:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)